test/e2e/go.mod

```diff
@@ -1,27 +1,31 @@
 module github.com/ovn-org/ovn-kubernetes/test/e2e
 
-go 1.22.0
+go 1.23.0
+
+toolchain go1.23.6
```

Comment:
Can't we just use go 1.23.6 so we don't have the toolchain thing ?

Reply:
This change is automated. Best not fight the automation. What is the problem with the toolchain thing?

Reply:
Fair enough